src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/SpecimenCharacteristicType.java

```diff
@@ -0,0 +1,57 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example SPC+TSP+:::BLOOD & URINE'
+ */
+@RequiredArgsConstructor
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class SpecimenCharacteristicType extends Segment {
+    private static final String KEY = "SPC";
+    private static final String QUALIFIER = "TSP";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+    private static final int THREE = 3;
```

Comment:
Could we rename this to describe the data being extracted from the EDIFACT string?
```suggestion
    private static final int CHARACTERISTIC_DETAILS_INDEX = 3;
```

Reply:
Fixed in 83df129755f827b8fba2bdeb86329a0d87300eed